app/models/user.rb

```diff
@@ -107,6 +107,28 @@ def effective_responsible_body
     responsible_body || school&.responsible_body
   end
 
+  def hybrid?
+    school_id && responsible_body_id
+  end
+
+  def hybrid_setup!
```

Comment:
no idea where to put this so I've shoved it here. probably not the right place...

Reply:
I would be tempted to push this out into a service.

Reply:
Ok. I think we can do that next time round. I've left everything in one place so it should be trivial to extract out